source/Cosmos.Core/AdvancedObject.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cosmos.Core;
+using IL2CPU.API;
+
+namespace Cosmos.Core
+{
+    /// <summary>
+    /// This Class is created for Add-ons methods to object by add this to base class
+    /// Use this class if you stuck on a prolem
```

Comment:
Uninformative.
Stuck on what problem?
Also - fix typos and capitals please.

Reply:
איזה שגיאת כתיב יש לי פה?

Reply:
for people who cant read hebrew "what is the spelling error I have here?"